### DISCUSSION TITLE
How to tune the dark mode of a website to my own preferences?

### DISCUSSION THREAD

**Discussion by oeloo** [darkreader/darkreader#13372] [Q&A]
Hi, how can I modify the Dark Reader dark mode configuration for a website, to improve it and then submit it to the community?
So that if the community likes it, they can add it to Dark Reader. Regards

**Comment by e-t-l** [comment]
To modify it for yourself, you can open the Dark Reader flyout and click Settings > Advanced > Dev Tools > {your preferred theme} Editor > Per Site Editor. There you can add any CSS tweaks you like. To my knowledge there isn't a community-maintained repository of theme mods, but if your modifications fix something that's currently broken in Dark Reader's default theme, then you could [fork the project](https://github.com/dimitrov-adrian/RcloneTray/fork) and submit a Pull Request for probably one of these files: https://github.com/darkreader/darkreader/tree/main/src/config (depending on which theme you're editing).

If your modifications aren't fixing a problem, and they're just tweaking the dark mode to better fit your preference, it will probably not be accepted, though.
